### DISCUSSION TITLE
Keyboard ignored temporarily - how do I stop this?

### DISCUSSION THREAD

**Discussion by Ro-87** [pqrs-org/Karabiner-Elements#3640] [Q&A]
Apologies if I am doing this wrong, I have never used GitHub before.

I have had Karabiner for a while to help with a faulty keyboard but it has stopped working! My backslash key presses itself regularly (internal fault, not a sticky button) so Karabiner is set to treat \ as vk_none.

But now every time it presses itself (whenever I press l, o, 9, . and sometimes m, p, n and others) Karabiner now says the keyboard is ignored temporarily until I press \. The info on Karabiner website says the temporarily ignore alert happens if "Your device sends unintented key up events regardless you do not touch the keys. (e.g., Swiftpoint ProPoint sends key up events of the music play key periodically.)" but this is exactly when I need Karabiner to work! Its only purpose is to disable \ when I **don't** press it.

How do I change this? I don't know how to code so will need very idiot-proof advice please. 
![A80D1C16-A15C-4A6B-85AA-84A759950D53_4_5005_c](https://github.com/pqrs-org/Karabiner-Elements/assets/150469546/07e0c458-ecdc-4d7c-9f11-84c902e78bd8)

**Comment by mystagog** [comment]
I have the same issue, help would be a blessing

I FIXED THE BUG BY LOWERING THE KEY REPEAT RATE IN THE OPTIONS

**Comment by hopscotch98** [comment]
Hi -- having the same exact issue here, except I've literally deleted the application from my mac and it's still happening every day. It's driving me crazy. Someone please help me get rid of it.

**Reply by mystagog** [reply]
Try going in the settings of your mac and change the keyboard sensitivity (you can type this in spotlight).
Then put the key repeat rate to the minimum. 
It doesn't completely solve the issue but it makes it appear much less frequently.
